src/context/context.ts

```diff
@@ -278,21 +278,50 @@ export class WorkflowContext<TInitialPayload = unknown> {
    * @returns call result (parsed as JSON if possible)
    */
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
-  public async call<TResult = unknown, TBody = unknown>(
+  public async call(
```

Comment:
Why did we remove the typing here? I think it's nice to have the option of `const res = call<T>()` instead of `const res = call() as T`

Reply:
to get rid of no-unnecessary-type-parameters warning

Reply:
We could also disable that rule for this page

Reply:
Yusuf was suggesting removing it, how about removing it for now and adding it later?

We can add it later but can't remove it later. I want to think about this a bit more because there is no generic in await request.json() for instance